src/components/homePage.component.js

```diff
@@ -181,25 +172,25 @@ export default function MyApp({ mode, theme, colorMode }) {
                       // Use a switch statement or if-else to determine the route based on the icon
                       switch (index) {
```

Comment:
seems like this `switch` statement is overkill, maybe something as simple as `navigate("/" + text.toLower());` would suffice?